src/elements/Toggle/Toggle.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Toggle
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Common from '../Common/Common';
+
+const Toggle = ({
+	checked,
+	value,
+	loading,
+	...other
+}) => {
+	const combinedClasses = classNames(
+		'ui-toggle',
+		other.classes,
+		checked && 'on',
```

Comment:
When using `classNames`, you can put an object in any argument position where the keys are the class names, and the values evaluate to truthy/falsy to turn those class names on. It's much more readable when you have long lists of classes to toggle:

```
classNames('ui-component', {
  'some-class': someCheck,
  on: checked,
  loading
})
```

Reply:
So like this:
```
	const combinedClasses = classNames(
		'ui-toggle',
		other.classes,
		{
			'on': checked,
			loading
		}
	);
```

Reply:
yessir!